cloud/main.js

```diff
@@ -141,8 +141,8 @@ Parse.Cloud.define("getChatRooms", function(request, response) {
         var chatRoomPromises = _.map(chatRoomIds, function(id) {
             return ChatRoom.get(id);
         });
-        var chatEventPromises = _.map(chatRoomIds, function(id) {
-            return ChatEvent.getMostRecentForChatRoom(id);
+        var chatEventPromises = _.map(aliases, function(alias) {
```

Comment:
Actually you can change this to _.map(aliases, ChatEvent.getMost...) and it'll work exactly the same but there's no need to change it for now